core/chains/evm/config/config.go

```diff
@@ -849,6 +853,58 @@ func (c *chainScopedConfig) BalanceMonitorEnabled() bool {
 	return c.defaultSet.balanceMonitorEnabled
 }
 
+// EvmEIP1559DynamicFees will send transactions with the 0x2 dynamic fee EIP-2718
+// type and gas fields when enabled
+func (c *chainScopedConfig) EvmEIP1559DynamicFees() bool {
+	val, ok := c.GeneralConfig.GlobalEvmEIP1559DynamicFees()
+	if ok {
+		c.logEnvOverrideOnce("EvmEIP1559DynamicFees", val)
+		return val
+	}
+	if c.persistedCfg.EvmEIP1559DynamicFees.Valid {
+		c.logPersistedOverrideOnce("EvmEIP1559DynamicFees", c.persistedCfg.EvmEIP1559DynamicFees.Bool)
+		return c.persistedCfg.EvmEIP1559DynamicFees.Bool
+	}
+	return c.defaultSet.eip1559DynamicFees
+}
+
+// EvmGasFeeCap is the fixed amount to set the fee cap on DynamicFee transactions
+// The recommended way to use it is to set this value to something very large
+// and control prices using the gas tip cap instead
+func (c *chainScopedConfig) EvmGasFeeCap() *big.Int {
+	return c.EvmMaxGasPriceWei()
```

Comment:
Are you suggesting adding a new variable to control this?

I'm not sure what that would add, since the effective meaning is identical. "This is the maximum I ever want to pay for gas" maps pretty much perfectly to the GasFeeCap.

Reply:
I created new env vars for all the others because they don't have a 1:1 mapping of meaning

Reply:
yeah thats what I was suggesting. I could see a nop thinking "I guess this MAX_GAS_PRICE doesn't have any effect since there are no gas "prices" with 1559". I think as long as its documented clearly its fine as is

Reply:
We definitely need to update the official docs once this is released.